src/SegmentedMsg.chpl

```diff
@@ -170,6 +170,40 @@ module SegmentedMsg {
     return new MsgTuple(repMsg, MsgType.NORMAL);
   }
 
+  proc getSegStringPropertyMsg(cmd: string, msgArgs: borrowed MessageArgs, st: borrowed SymTab): MsgTuple throws {
+    var pn = Reflection.getRoutineName();
+    const property = msgArgs.getValueOf("property");
+    const name = msgArgs.getValueOf("obj");
+
+    var genSym = toGenSymEntry(st.lookup(name));
+
+    if genSym.dtype != DType.Strings{
+      var errorMsg = notImplementedError(pn, "%s".format(dtype2str(genSym.dtype)));
+      smLogger.error(getModuleName(),getRoutineName(),getLineNumber(),errorMsg);                      
+      return new MsgTuple(errorMsg, MsgType.ERROR);
+    }
+    
+    var ssentry = toSegStringSymEntry(genSym);
+    var rname = st.nextName();
+    select property{
+      when ("mybytes"){
+        st.addEntry(rname, ssentry.bytesEntry);
+      }
+      when ("myoffsets"){
+        st.addEntry(rname, ssentry.offsetsEntry);
+      }
```

Comment:
this is completely stylistic and there's no need to change but u don't need the parens here

Reply:
Updated